06-building-a-program-start-to-finish/writing-tests/output/test.js

```diff
@@ -0,0 +1,133 @@
+const InvertedIndex = require('../src/inverted-index.js');
+const books = require('./books');
+
+describe('Inverted index Suite', () => {
+   // Create an instance of the Index class
+  const newIndex = new InvertedIndex();
+  const emptyBook = [];
+  const sampleSentence = 'As &you can see here, y*ou, #@have ==defined *the function';
+  const multipleSearch = 'Coverage lines for you';
+  beforeEach(() => {
```

Comment:
I did not know about beforeAll function before now, but I feel beforeAll should be better since it is an index that can be accessed by all the 'it' block.